editoast/editoast_schemas/src/rolling_stock/train_category.rs

```diff
@@ -45,3 +46,12 @@ impl From<TrainCategories> for Vec<Option<TrainCategory>> {
         categories.0.into_iter().map(Some).collect()
     }
 }
+
+#[derive(Clone, Debug, PartialEq, Deserialize, Serialize, ToSchema)]
+pub struct SubCategory {
+    pub id: i64,
```

Comment:
The schema should not contain an ID, right?
In addition, if we have a code that should be unique, I don't think it's required to have it in our DB model, too.

Reply:
Yes, we can use code as the primary key.
I’ve addressed the comments, @flomonster.

Reply:
This is a schema, not a module. Primary keys do not make sense in that context. 

However @flomonster using natural primary keys can only bring trouble on the long run and that's something we avoided using until now. What's the benefit in this case?

Reply:
True, we did the same for rolling stock. It's just an unused field and I don't see why it's that important for the long run.

Since it's a question for the implementation PR, I'm closing this conversation.